cmd/probe/src/lib.rs

```diff
@@ -241,10 +241,12 @@ fn probecmd(context: &mut humility::ExecutionContext) -> Result<()> {
                 .and_then(|_| core.read_reg(Register::Arm(ARMRegister::PC)))
                 .and_then(|val| {
                     core.step()?;
-                    Ok(val)
+                    Ok(val as u32)
                 })
                 .and_then(|val| {
-                    if core.read_reg(Register::Arm(ARMRegister::PC))? == val {
+                    if core.read_reg(Register::Arm(ARMRegister::PC))? as u32
```

Comment:
Why are we changing ARM register code - is this a fallout from the refactoring elsewhere?

Reply:
To support 64bit register reads for rv64, I had to change the interface for the `[Core](https://github.com/rivosinc/humility/pull/19/files#diff-9adbbcf92e22278f3242e9863687df99dac2db69f490b9f288107176119a6656)` to use u64.  The trait has to be shared by arm, rv32, and rv64.  This is just fallout of that API change.